src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -83,6 +95,20 @@ public void start(Future<Void> startFuture) {
       handlers.put(END_PATRON_SESSION, injector.getInstance(EndPatronSessionHandler.class));
     }
 
+    JsonObject crOptionsJson = config().getJsonObject("tenantConfigRetrieverOptions");
+    ConfigRetrieverOptions crOptions = new ConfigRetrieverOptions(crOptionsJson);
+    ConfigRetriever configRetriever = ConfigRetriever.create(vertx, crOptions);
+
+    configRetriever.getConfig(ar -> {
```

Comment:
We probably want to `configRetriever.close()` when the verticle is stopped (below).

Reply:
Yes, I'll add that.